File: 2022.1/Programação Orientada a Objetos/lista1java/Ex1.java
public class Ex1{
    public static void main(String args[]) throws Exception{
        System.out.print("Digite o valor de x: ");
        double x = EntradaTeclado.leDouble();
        System.out.print("Digite o chute inicial: ");
        double x1 = EntradaTeclado.leDouble();
        double x0;

        do{
            x0 = x1;
            x1 = (x0 + x/x0)/2;
        }
        while((Math.abs(x1 - x0)) > 0.00000001);

        System.out.println(x1);
    }
}

File: 2022.1/Programação Orientada a Objetos/lista1python/Ex1.py
import math

x = float(input("Digite um número: "))
x0 = float(input("Digite um chute inicial: "))
xi = x0

xi = (x0 + x/x0)/2

while math.fabs(xi - x0) > 0.00000001:
    x0 = xi
    xi = (x0 + x/x0)/2

print(xi);